loom/graphics/gfxTexture.cpp

```diff
@@ -368,15 +364,49 @@ TextureInfo *Texture::load(uint8_t *data, uint16_t width, uint16_t height, Textu
 
             mipLevel++;
         }
+        lmLogInfo(gGFXTextureLogGroup, "Generated mipmaps in %d ms", platform_getMilliseconds() - time);
         if (mipData != (uint32_t*) data) lmFree(NULL, mipData);
     } else {
         tinfo.clampOnly = true;
         tinfo.mipmaps = false;
-        lmLogWarn(gGFXTextureLogGroup, "Non-power-of-two textures not fully supported by device, consider using a power-of-two texture size")
-    }
+		if (!supportsFullNPOT) lmLogWarn(gGFXTextureLogGroup, "Non-power-of-two textures not fully supported by device, consider using a power-of-two texture size")
+    }
+
+    if (newTexture && tinfo.renderTarget)
```

Comment:
Not sure what you mean? The code above it sets up mip mapping and this code sets up a framebuffer. You could probably jam them together, but I think it looks cleaner this way.